src/js/components/List/__tests__/List-test.js

```diff
@@ -21,6 +21,7 @@ describe('List', () => {
     const { container, getByText } = render(
       <Grommet>
         <List
+          aria-label="List"
```

Comment:
Is their value to adding Jest tests for the various onOrder functionality? Thinking about:
- keyboard interactions
- getByText, mouseover up button, click button, confirm reorder occurred ...
- something to provide developer feedback if they are combining onOrder with paginate and/or onClickItem (test would confirm the feedback, console warning or the like would need to be implemented) --> this is documented, but does something like this improve the developer experience? --> is this overthinking?

Reply:
I implemented the tests for keyboard and mouse onOrder functionality.
I didn't put in a warning for the onClickItem since it seems reasonable to still specify onClickItem even though you temporarily specify onOrder to let it be reordered (e.g. the example in the figma)